### DISCUSSION TITLE
基于Lemon Squeezy开发你的全球可用的会员功能 | J实验室

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/new-blog#2] [General]
# 基于Lemon Squeezy开发你的全球可用的会员功能 | J实验室

全栈工程师，Next.js开源手艺人，掘金签约作者，折腾才有未来

https://weijunext.com/article/integrate-lemonsqueezy-api

**Comment by lvxianchao** [comment]
出海小白想请教一下站长有用过 Paddle 吗？对比 Paddle 如何？我现在哪个都没用过

**Reply by weijunext** [reply]
没用过。这几个支付渠道网上有很多人测评了，你可以谷歌搜一下

**Reply by lvxianchao** [reply]
是，我觉得网上搜的没有你写的靠谱

**Comment by gouyuanyang** [comment]
想请教下，为什么订阅的时候还会发送subscription_updated事件，这个很疑惑
